andela/trime
# Link to projects from dashboard

### What does this PR do?
Makes the projects' titles in the the project table for each in the organization a link to the project page.

### Description of Task to be completed?
Make the projects' titles in the the project table for each in the organization a link to the project page.

### How should this be manually tested?
Go to the dashboard. Create an organization if you don't have any. If none of your organizations have a project, create a project for one of them. You should see each organization include a list of the project in that organization and clicking on the a projects title takes you to that projects page.

### What are the relevant pivotal tracker stories?
[Link to project from the dashboard](https://www.pivotaltracker.com/story/show/128309653)
